packages/clerk_flutter/lib/src/clerk_auth_state.dart

```diff
@@ -71,7 +71,10 @@ class ClerkAuthState extends clerk.Auth with ChangeNotifier {
   static const _kSsoRouteName = 'clerk_sso_popup';
 
   @override
-  void update() => notifyListeners();
+  void update() {
```

Comment:
super class clerk.Auth update() function should have `@mustCallSuper` annotation